odoo/addons/base/module/wizard/base_module_upgrade.py

```diff
@@ -79,4 +79,5 @@ def upgrade_module(self):
 
     @api.multi
     def config(self):
+        # pylint: disable=next-method-called
```

Comment:
Either put all line pragmas on the previous line or put them all on the same line as what they suppress, otherwise it's confusing.